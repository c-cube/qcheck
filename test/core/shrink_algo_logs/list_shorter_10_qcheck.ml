open QCheck

let list_shorter_10 =
  Test.make ~name:"lists shorter than 10"
    (list small_int) (Log.shrinks Print.(list int) @@ fun xs -> List.length xs < 10)
;;
try Test.check_exn list_shorter_10 with Test.Test_fail _ -> ()
