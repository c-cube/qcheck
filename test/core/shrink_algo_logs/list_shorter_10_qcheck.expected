holds [9; 3; 94; 3]
fails [8; 81; 5; 3; 61; 1; 8; 3; 8; 8; 13; 6; 8; 6; 8; 2; 4; 9; 0; 55; 21; 6; 6; 0; 6; 1; 5; 5; 9; 20; 3; 7; 86; 2; 38; 2; 9; 0; 2; 5; 1; 8; 5; 4; 1; 4; 3; 48; 0; 8; 80; 4; 0; 1; 6; 0; 6; 4; 9; 4; 39; 4; 7; 6; 23; 20; 47; 2; 5; 3; 1; 7; 14; 1; 2; 94; 0; 2; 6; 38; 5; 18; 5; 31; 0; 5; 4; 30; 2; 8; 19; 0; 8; 1; 7]
fails [0; 8; 80; 4; 0; 1; 6; 0; 6; 4; 9; 4; 39; 4; 7; 6; 23; 20; 47; 2; 5; 3; 1; 7; 14; 1; 2; 94; 0; 2; 6; 38; 5; 18; 5; 31; 0; 5; 4; 30; 2; 8; 19; 0; 8; 1; 7]
fails [14; 1; 2; 94; 0; 2; 6; 38; 5; 18; 5; 31; 0; 5; 4; 30; 2; 8; 19; 0; 8; 1; 7]
fails [0; 5; 4; 30; 2; 8; 19; 0; 8; 1; 7]
holds [19; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 5; 8; 1; 7]
holds [0; 5; 4; 1; 7]
holds [0; 5; 4; 30; 7]
holds [0; 5; 4; 30; 2]
holds [30; 2; 8; 19; 0; 8; 1; 7]
holds [0; 2; 8; 19; 0; 8; 1; 7]
holds [0; 5; 8; 19; 0; 8; 1; 7]
holds [0; 5; 4; 19; 0; 8; 1; 7]
holds [0; 5; 4; 30; 0; 8; 1; 7]
holds [0; 5; 4; 30; 2; 8; 1; 7]
holds [0; 5; 4; 30; 2; 8; 1; 7]
holds [0; 5; 4; 30; 2; 8; 19; 7]
holds [0; 5; 4; 30; 2; 8; 19; 0]
fails [5; 4; 30; 2; 8; 19; 0; 8; 1; 7]
holds [19; 0; 8; 1; 7]
holds [5; 0; 8; 1; 7]
holds [5; 4; 8; 1; 7]
holds [5; 4; 30; 1; 7]
holds [5; 4; 30; 2; 7]
holds [5; 4; 30; 2; 8]
holds [30; 2; 8; 19; 0; 8; 1; 7]
holds [5; 2; 8; 19; 0; 8; 1; 7]
holds [5; 4; 8; 19; 0; 8; 1; 7]
holds [5; 4; 30; 19; 0; 8; 1; 7]
holds [5; 4; 30; 2; 0; 8; 1; 7]
holds [5; 4; 30; 2; 8; 8; 1; 7]
holds [5; 4; 30; 2; 8; 19; 1; 7]
holds [5; 4; 30; 2; 8; 19; 0; 7]
holds [5; 4; 30; 2; 8; 19; 0; 8]
holds [4; 30; 2; 8; 19; 0; 8; 1; 7]
holds [5; 30; 2; 8; 19; 0; 8; 1; 7]
holds [5; 4; 2; 8; 19; 0; 8; 1; 7]
holds [5; 4; 30; 8; 19; 0; 8; 1; 7]
holds [5; 4; 30; 2; 19; 0; 8; 1; 7]
holds [5; 4; 30; 2; 8; 0; 8; 1; 7]
holds [5; 4; 30; 2; 8; 19; 8; 1; 7]
holds [5; 4; 30; 2; 8; 19; 0; 1; 7]
holds [5; 4; 30; 2; 8; 19; 0; 8; 7]
holds [5; 4; 30; 2; 8; 19; 0; 8; 1]
fails [0; 4; 30; 2; 8; 19; 0; 8; 1; 7]
holds [19; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 4; 8; 1; 7]
holds [0; 4; 30; 1; 7]
holds [0; 4; 30; 2; 7]
holds [0; 4; 30; 2; 8]
holds [30; 2; 8; 19; 0; 8; 1; 7]
holds [0; 2; 8; 19; 0; 8; 1; 7]
holds [0; 4; 8; 19; 0; 8; 1; 7]
holds [0; 4; 30; 19; 0; 8; 1; 7]
holds [0; 4; 30; 2; 0; 8; 1; 7]
holds [0; 4; 30; 2; 8; 8; 1; 7]
holds [0; 4; 30; 2; 8; 19; 1; 7]
holds [0; 4; 30; 2; 8; 19; 0; 7]
holds [0; 4; 30; 2; 8; 19; 0; 8]
holds [4; 30; 2; 8; 19; 0; 8; 1; 7]
holds [0; 30; 2; 8; 19; 0; 8; 1; 7]
holds [0; 4; 2; 8; 19; 0; 8; 1; 7]
holds [0; 4; 30; 8; 19; 0; 8; 1; 7]
holds [0; 4; 30; 2; 19; 0; 8; 1; 7]
holds [0; 4; 30; 2; 8; 0; 8; 1; 7]
holds [0; 4; 30; 2; 8; 19; 8; 1; 7]
holds [0; 4; 30; 2; 8; 19; 0; 1; 7]
holds [0; 4; 30; 2; 8; 19; 0; 8; 7]
holds [0; 4; 30; 2; 8; 19; 0; 8; 1]
fails [0; 0; 30; 2; 8; 19; 0; 8; 1; 7]
holds [19; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 30; 1; 7]
holds [0; 0; 30; 2; 7]
holds [0; 0; 30; 2; 8]
holds [30; 2; 8; 19; 0; 8; 1; 7]
holds [0; 2; 8; 19; 0; 8; 1; 7]
holds [0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 30; 19; 0; 8; 1; 7]
holds [0; 0; 30; 2; 0; 8; 1; 7]
holds [0; 0; 30; 2; 8; 8; 1; 7]
holds [0; 0; 30; 2; 8; 19; 1; 7]
holds [0; 0; 30; 2; 8; 19; 0; 7]
holds [0; 0; 30; 2; 8; 19; 0; 8]
holds [0; 30; 2; 8; 19; 0; 8; 1; 7]
holds [0; 30; 2; 8; 19; 0; 8; 1; 7]
holds [0; 0; 2; 8; 19; 0; 8; 1; 7]
holds [0; 0; 30; 8; 19; 0; 8; 1; 7]
holds [0; 0; 30; 2; 19; 0; 8; 1; 7]
holds [0; 0; 30; 2; 8; 0; 8; 1; 7]
holds [0; 0; 30; 2; 8; 19; 8; 1; 7]
holds [0; 0; 30; 2; 8; 19; 0; 1; 7]
holds [0; 0; 30; 2; 8; 19; 0; 8; 7]
holds [0; 0; 30; 2; 8; 19; 0; 8; 1]
fails [0; 0; 0; 2; 8; 19; 0; 8; 1; 7]
holds [19; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 2; 7]
holds [0; 0; 0; 2; 8]
holds [0; 2; 8; 19; 0; 8; 1; 7]
holds [0; 2; 8; 19; 0; 8; 1; 7]
holds [0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 2; 0; 8; 1; 7]
holds [0; 0; 0; 2; 8; 8; 1; 7]
holds [0; 0; 0; 2; 8; 19; 1; 7]
holds [0; 0; 0; 2; 8; 19; 0; 7]
holds [0; 0; 0; 2; 8; 19; 0; 8]
holds [0; 0; 2; 8; 19; 0; 8; 1; 7]
holds [0; 0; 2; 8; 19; 0; 8; 1; 7]
holds [0; 0; 2; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 2; 19; 0; 8; 1; 7]
holds [0; 0; 0; 2; 8; 0; 8; 1; 7]
holds [0; 0; 0; 2; 8; 19; 8; 1; 7]
holds [0; 0; 0; 2; 8; 19; 0; 1; 7]
holds [0; 0; 0; 2; 8; 19; 0; 8; 7]
holds [0; 0; 0; 2; 8; 19; 0; 8; 1]
fails [0; 0; 0; 1; 8; 19; 0; 8; 1; 7]
holds [19; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 1; 8]
holds [0; 1; 8; 19; 0; 8; 1; 7]
holds [0; 1; 8; 19; 0; 8; 1; 7]
holds [0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 1; 0; 8; 1; 7]
holds [0; 0; 0; 1; 8; 8; 1; 7]
holds [0; 0; 0; 1; 8; 19; 1; 7]
holds [0; 0; 0; 1; 8; 19; 0; 7]
holds [0; 0; 0; 1; 8; 19; 0; 8]
holds [0; 0; 1; 8; 19; 0; 8; 1; 7]
holds [0; 0; 1; 8; 19; 0; 8; 1; 7]
holds [0; 0; 1; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 1; 19; 0; 8; 1; 7]
holds [0; 0; 0; 1; 8; 0; 8; 1; 7]
holds [0; 0; 0; 1; 8; 19; 8; 1; 7]
holds [0; 0; 0; 1; 8; 19; 0; 1; 7]
holds [0; 0; 0; 1; 8; 19; 0; 8; 7]
holds [0; 0; 0; 1; 8; 19; 0; 8; 1]
fails [0; 0; 0; 0; 8; 19; 0; 8; 1; 7]
holds [19; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 8]
holds [0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 8; 8; 1; 7]
holds [0; 0; 0; 0; 8; 19; 1; 7]
holds [0; 0; 0; 0; 8; 19; 0; 7]
holds [0; 0; 0; 0; 8; 19; 0; 8]
holds [0; 0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 8; 19; 0; 8; 1; 7]
holds [0; 0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 0; 8; 0; 8; 1; 7]
holds [0; 0; 0; 0; 8; 19; 8; 1; 7]
holds [0; 0; 0; 0; 8; 19; 0; 1; 7]
holds [0; 0; 0; 0; 8; 19; 0; 8; 7]
holds [0; 0; 0; 0; 8; 19; 0; 8; 1]
fails [0; 0; 0; 0; 0; 19; 0; 8; 1; 7]
holds [19; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 19; 1; 7]
holds [0; 0; 0; 0; 0; 19; 0; 7]
holds [0; 0; 0; 0; 0; 19; 0; 8]
holds [0; 0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 0; 19; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 19; 8; 1; 7]
holds [0; 0; 0; 0; 0; 19; 0; 1; 7]
holds [0; 0; 0; 0; 0; 19; 0; 8; 7]
holds [0; 0; 0; 0; 0; 19; 0; 8; 1]
fails [0; 0; 0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 8; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 8]
holds [0; 0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 0; 8; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 8; 7]
holds [0; 0; 0; 0; 0; 0; 0; 8; 1]
fails [0; 0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 1; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 1]
fails [0; 0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 7]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
fails [0; 0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
holds [0; 0; 0; 0; 0; 0; 0; 0; 0]
